index.html

```diff
@@ -43,6 +43,139 @@ <h3 class="title dtitle">I'm a software developer.</h3>
+                <div class="card">
+                    <div class="card-image"></div>
+                    <div class="card-text">
+                        <p class="card-title">Multi-Post Stories Gain + Glory</p>
+                        <ul class="card-tools-list">
+                            <li class="card-tool">Ruby on rails</li>
+                            <li class="card-tool"> CSS</li>
+                            <li class="card-tool">Javascript</li>
+                            <li class="card-tool"> HTML</li>
+                        </ul>
+                        <button type="button" class="see-project">See project</button>
+                    </div>
+                </div>
+                <div class="card">
+                    <div class="card-image"></div>
+                    <div class="card-text">
+                        <p class="card-title">Multi-Post Stories Gain + Glory</p>
+                        <ul class="card-tools-list">
+                            <li class="card-tool">Ruby on rails</li>
+                            <li class="card-tool"> CSS</li>
+                            <li class="card-tool">Javascript</li>
+                            <li class="card-tool"> HTML</li>
+                        </ul>
+                        <button type="button" class="see-project">See project</button>
+                    </div>
+                </div>
+                <div class="card">
+                    <div class="card-image"></div>
+                    <div class="card-text">
+                        <p class="card-title">Multi-Post Stories Gain + Glory</p>
+                        <ul class="card-tools-list">
+                            <li class="card-tool">Ruby on rails</li>
+                            <li class="card-tool"> CSS</li>
+                            <li class="card-tool">Javascript</li>
+                            <li class="card-tool"> HTML</li>
+                        </ul>
+                        <button type="button" class="see-project">See project</button>
+                    </div>
+                </div>
+                <div class="card">
+                    <div class="card-image"></div>
+                    <div class="card-text">
+                        <p class="card-title">Multi-Post Stories Gain + Glory</p>
+                        <ul class="card-tools-list">
+                            <li class="card-tool">Ruby on rails</li>
+                            <li class="card-tool"> CSS</li>
+                            <li class="card-tool">Javascript</li>
+                            <li class="card-tool"> HTML</li>
+                        </ul>
+                        <button type="button" class="see-project">See project</button>
+                    </div>
+                </div>
+                
+            </div>
+        </section>
+
+        <section class="about-me">
+            <div class="text">
+                <div class="primary-text">About me</div>
```

Comment:
- [ ] please make use of one of the various headings asides the `<h1>` (which is to be used once in the web page). To display the about me text  you can select from `<h2>, <h3>` etc :+1: